### DISCUSSION TITLE
'Multiple clients signing in withing a single realm' support by 'keycloak-js adapter'

### DISCUSSION THREAD

**Discussion by antonfil** [keycloak/keycloak#17089] [Q&A]
Does 'keycloak-js adapter' (https://www.keycloak.org/docs/latest/securing_apps/index.html#_javascript_adapter) support signing into multiple clients within a single realm from single web application concurrently? Then I need to get 'access tokens' separately from these clients.

**Comment by jonkoops** [comment]
It does not support this at the moment, no. In the future we might accommodate this use-case, but I don't expect this will change any time soon.
